docs/using-the-compiler.rst

```diff
@@ -378,12 +378,11 @@ Input Description
         {
           // Choose which model checker engine to use: all (default), bmc, chc, none.
           "engine": "chc",
-          // Choose which targets should be checked: all (default), constantCondition,
+          // Choose which targets should be checked: constantCondition,
           // underflow, overflow, divByZero, balance, assert, popEmptyArray, outOfBounds.
+          // If the option is not given all targets are checked by default.
```

Comment:
In other options we use `"*"` for all, that may be an option here too?

Reply:
I was going for `empty=all`. Would you have "*" plus that?

Reply:
@axic In the CLI option one of the possible values is the explicit "all". Would it be preferable to have "*" or "all" here?

Reply:
I am not sure for which other options we have "*", but if we have it, maybe better to stay consistent and use "*" instead of "all"?

Reply:
We have `*` for bytecode output selection. I don't like that for the smtchecker targets though, because `all` is the default in some cases, which is different from the bytecode stuff. So for the smtchecker my favorite for these targets is still "empty". If we also want an explicit version of all, the options are:
- `all`: this is my favorite because it works for both JSON and CLI, but is inconsistent with bytecode output selection which uses `*`
- `*`: is consistent with bytecode but imo it doesn't work for CLI: `--model-checker-targets *` feels rather weird

Reply:
Sounds good to me to go with `all` in that case, if @axic agrees